crates/sncast/tests/e2e/deploy.rs

```diff
@@ -91,56 +94,57 @@ async fn test_happy_case(class_hash: Felt, account_type: AccountType) {
     assert!(matches!(receipt, Deploy(_)));
 }
 
-#[test_case(Some("99999999999999999"), None, None; "max_fee")]
-#[test_case(None, Some("999"), None; "max_gas")]
-#[test_case(None, None, Some("999999999999"); "max_gas_unit_price")]
-#[test_case(None, None, None; "none")]
-#[test_case(Some("999999999999999"), None, Some("999999999999"); "max_fee_max_gas_unit_price")]
-#[test_case(None, Some("999"), Some("999999999999"); "max_gas_max_gas_unit_price")]
-#[test_case(Some("999999999999999"), Some("999"), None; "max_fee_max_gas")]
-#[tokio::test]
-async fn test_happy_case_different_fees(
-    max_fee: Option<&str>,
-    max_gas: Option<&str>,
-    max_gas_unit_price: Option<&str>,
-) {
-    let tempdir = create_and_deploy_oz_account().await;
-
-    let mut args = vec![
-        "--accounts-file",
-        "accounts.json",
-        "--account",
-        "my_account",
-        "--int-format",
-        "--json",
-        "deploy",
-        "--url",
-        URL,
-        "--class-hash",
-        MAP_CONTRACT_CLASS_HASH_SEPOLIA,
-        "--salt",
-        "0x2",
-        "--unique",
-    ];
-    let options = [
-        ("--max-fee", max_fee),
-        ("--max-gas", max_gas),
-        ("--max-gas-unit-price", max_gas_unit_price),
-    ];
-
-    for &(key, value) in &options {
-        if let Some(val) = value {
-            args.append(&mut vec![key, val]);
-        }
-    }
-
-    let snapbox = runner(&args).current_dir(tempdir.path());
-    let output = snapbox.assert().success().get_output().stdout.clone();
-    let hash = get_transaction_hash(&output);
-    let receipt = get_transaction_receipt(hash).await;
-
-    assert!(matches!(receipt, Deploy(_)));
-}
+// TODO(#3090)
```

Comment:
Uncomment and ignore

Reply:
Refactored it and linked #3100 . The test case with max fee has been commented out as it's not possible to ignore a specific one.